lesson-3/object_finder.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ObjectFinder
+  private
+
+  def find(array, attr_name, input)
+    array.find { |e| e.instance_variable_get("@#{attr_name.to_sym}") == input }
```

Comment:
Can we use getter instead of `instance_variable_get`?

Reply:
Metaprogramming...
Train -> number
Station -> name
Or can I write like this e.attr_name?

Reply:
> Metaprogramming...
> Train -> number
> Station -> name
> Or can I write like this e.attr_name?

You can write a new getter

Reply:
Something like alias for number and name?